### DISCUSSION TITLE
Can I directly download the JSON schema for `deno.jsonc`?

### DISCUSSION THREAD

**Discussion by David-Else** [denoland/deno#13091] [Q&A] [answered]
It is not available on https://www.schemastore.org/json/ , is it available somewhere in the same format as https://json.schemastore.org/tsconfig.json ? 

This would mean it could be used outside of VS Code, for example using the same JSON language server from VS Code via its own LSP in Neovim:
```
        schemas = {
          {
            fileMatch = { 'package.json' },
            url = 'https://json.schemastore.org/package.json',
          },
        },
```

**Comment by kitsonk** [comment, marked as answer]
It is available at https://github.com/denoland/deno/blob/main/cli/schemas/config-file.v1.json.

vscode_deno uses it from https://raw.githubusercontent.com/denoland/deno/main/cli/schemas/config-file.v1.json perfectly fine.

**Comment by David-Else** [comment]
Thanks! For Neovim add https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#jsonls and:
```
  jsonls = {
    settings = {
      json = {
        schemas = {
          {
            fileMatch = { 'deno.json' },
            url = 'https://raw.githubusercontent.com/denoland/deno/main/cli/schemas/config-file.v1.json',
          },
        },
      },
    },
  },
```
For `deno.jsonc` we need to add `jsonc` to the filetypes of `jsonls`.
